dashboard/test/integration/db_query_test.rb

```diff
@@ -20,7 +20,7 @@ def setup
       level: level,
       level_source: create(:level_source, level: level)
 
-    assert_cached_queries(18) do
+    assert_cached_queries(25) do
```

Comment:
So after a bit of tweaking, this is the only remaining query count that increases with this change.

Note, however, that this test was recently increased from 13 to 18 in https://github.com/code-dot-org/code-dot-org/pull/41420/files#diff-e5cda77c673c69979f0acdd7d326c708c2c051f612c80ebfdcf072fb318d172a, so this jump to 25 represents a near-doubling of the original query count. It's not clear to me how much of a concern that is.

Reply:
@code-dot-org/infrastructure any thoughts on this query increase?

Reply:
We can use NewRelic, AWS Performance Insights, and the Percona Monitoring “Query Analytics” tool to assess the impact of the current query and predict the impact of this proposed query change. Setup 15 minutes with me tomorrow to review how to do this?

Reply:
That'd be great! I'd love to learn a bit about those tools in general